monai/transforms/post/array.py

```diff
@@ -13,6 +13,7 @@
 https://github.com/Project-MONAI/MONAI/wiki/MONAI_Design
```

Comment:
maybe add a comment here that this postprocessing module assumes shape `[batch, channels, (optional spatial dimensions)]`

also, [`write_nifti`](https://github.com/Project-MONAI/MONAI/blob/master/monai/data/nifti_writer.py#L19) `write_png` are also postprocessings, shall we move them here? (perhaps need to think about this, not a high priority task for now)

Reply:
updated in the latest patch.
`write_nifiti` can be also used by pre-processing transform, to save some transform result.
`save_nifiti` can be really post_transforms I feel.
You can submit an task issue for that I think.
Thanks.